display_list/display_list_benchmarks.cc

```diff
@@ -32,6 +33,45 @@ std::unique_ptr<CanvasProvider> CreateCanvasProvider(BackendType backend_type) {
   return nullptr;
 }
 
+SkPaint GetPaintForRun(unsigned attributes) {
+  SkPaint paint;
+
+  if (attributes & kStrokedStyle_Flag && attributes & kFilledStyle_Flag) {
+    paint.setStyle(SkPaint::kStrokeAndFill_Style);
```

Comment:
Flutter doesn't have strokeAndFill, but it can't hurt to test it anyway as DL will accept that style.